mono/utils/mono-threads-windows.c

```diff
@@ -131,13 +121,14 @@ mono_threads_suspend_begin_async_resume (MonoThreadInfo *info)
 		context.ContextFlags = CONTEXT_INTEGER | CONTEXT_CONTROL;
 		res = SetThreadContext (handle, &context);
 		if (!res) {
-			CloseHandle (handle);
 			return FALSE;
 		}
 	}
+#else
+	g_assert(0 && "Not implemented due to lack of SetThreadContext");	
```

Comment:
`g_error("Not implemented due to lack of SetThreadContext");`

Reply:
I'll correct this.

Reply:
That's wierd. What does CoreCLR do? I guess they don't run on UWP. Ask for SetThreadContext to be put back?

Reply:
No idea.

Reply:
I've fixed the assert at 06fa3967b744b426910fd14f5003ca1c8bd61875.